util/config/config.go

```diff
@@ -175,6 +177,11 @@ type Configuration struct {
 }
 
 func Init() error {
+	syncBatchWindowSize := uint32(memory.TotalMemory()*MaxSyncMemoryUsagePercent/100/MaxBlockSize) / Parameters.SyncBlocksBatchSize
+	if syncBatchWindowSize > 0 {
+		Parameters.SyncBatchWindowSize = syncBatchWindowSize
```

Comment:
It will be overwrote by config.json.

Reply:
That's intended, this only changes the default value, user can always override it